app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command_name, file_name, mv_file = command.split()
+        directories, file_name_out = os.path.split(mv_file)
+        if command_name == "mv" and os.path.isfile(file_name):
+            if directories:
+                os.makedirs(directories, exist_ok=True)
+            os.rename(file_name, mv_file)
```

Comment:
If you want cross-platform overwriting of the destination, use [replace()](https://docs.python.org/3/library/os.html?highlight=os%20rename#os.replace)